adr/ADR-20.md

```diff
@@ -333,21 +333,28 @@ GetFile(name string, file string)
 
 **GetInfo**
 
-GetInfo will retrieve the current information for the object, including the info of a Deleted object.
+GetInfo will retrieve the current information for the object. 
+* Do not return info for deleted objects, except with optional convenience methods. 
 
 ```
 GetInfo(name string) -> ObjectInfo
 ```
 
+Optional/Convenience examples:
+
+```
+GetInfo(name string, includingDeleted bool) -> ObjectInfo
+GetInfo(name string, opts ...WatchOpt) -> ObjectInfo
+```
+
 **UpdateMeta**
 
 UpdateMeta will update **some** metadata for the object.
 * Only the name, description and headers can be updated.
 * Objects, Links and Bucket Links are all allowed to be updated.
-* It is okay to change the name if the name does not exist.
-* It is okay to change the name to that of an existing but deleted object.
-* It is an error to change the name to that of an existing but not deleted object.
 * It is an error to update metadata for a deleted object.
+* It is an error to change the name to that of an existing object.
```

Comment:
It is an error to rename an object to the name of an existing object

Reply:
fixed